csharp/Platform.Collections/Segments/Walkers/AllSegmentsWalkerBase.cs

```diff
@@ -3,18 +3,14 @@
 namespace Platform.Collections.Segments.Walkers
 {
     /// <summary>
-    /// <para>
-    /// Represents the all segments walker base.
-    /// </para>
-    /// <para></para>
+    /// <para>Represents the base abstract class for walkers on all elements.</para>
+    /// <para>Представляет базовый абстрактный класс для проходчиков по всем элементам.</para>
     /// </summary>
     public abstract class AllSegmentsWalkerBase
     {
         /// <summary>
-        /// <para>
-        /// The default minimum string segment length.
-        /// </para>
-        /// <para></para>
+        /// <para>Gets default minimum string segment length.</para> 
```

Comment:
```suggestion
        /// <para>Gets a default minimum length of string segment.</para> 
```
More readable, and `a` added.

Reply:
Hold my tea